docs/rules/no-ember-super-in-es-classes.md

```diff
@@ -0,0 +1,34 @@
+## `this._super` is not allowed in ES class methods
+
+### Rule name: `no-ember-super-in-es-classes`
```

Comment:
Names are hard 😹

Reply:
No kidding. Wide open if you have any other suggestions 😁